### PR TITLE
Complete first draft of press release

### PR DESCRIPTION
**Pull request by ajkitson**

### CODE REVIEW

**Comment by brianpchsu** [comment]
Nice job.

File: _PRESS-RELEASE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Project Name #
+# Ribbet #
```
Comment (rjatkinson2) [inline _PRESS-RELEASE.md]: Fix spelling

File: _PRESS-RELEASE.md [hunk 2]
```diff
@@ -17,28 +17,32 @@ Once the project moves into development, the press release can be used as a touc
...
-  > Describe how your product elegantly solves the problem.
+  We turn everyone's phones and laptops into a mic.
 
 ## Quote from You ##
-  > A quote from a spokesperson in your company.
+  "Is this thing on? With Ribbit, it's always on. I'm always on." - SungMin
 
 ## How to Get Started ##
-  > Describe how easy it is to get started.
+  Visit ribbit.io, create a room, and invite your audience to join you.
```
Comment (rjatkinson2) [inline _PRESS-RELEASE.md]: We can't use ribbit.io.

File: _PRESS-RELEASE.md [hunk 3]
```diff
@@ -17,28 +17,32 @@ Once the project moves into development, the press release can be used as a touc
...
 ## How to Get Started ##
-  > Describe how easy it is to get started.
+  Visit ribbit.io, create a room, and invite your audience to join you. 
 
 ## Customer Quote ##
-  > Provide a quote from a hypothetical customer that describes how they experienced the benefit.
+  "It makes talking in lecture feel like a lunch table converation" - satisfied customer #1218817 and counting
 
 ## Closing and Call to Action ##
   > Wrap it up and give pointers where the reader should go next.
```
Comment (rjatkinson2) [inline _PRESS-RELEASE.md]: Need this.

**Pull request merged**